state/runtime/evm/instructions_test.go

```diff
@@ -494,9 +501,14 @@ func TestNot(t *testing.T) {
 	s, closeFn := getState(&chain.ForksInTime{})
 	defer closeFn()
 
+	firstValue, ok := new(big.Int).SetString("115792089237316195423570985008687907853269984665640564039457584007913129639935", 10)
+	require.True(t, ok)
+	secondValue, ok := new(big.Int).SetString("115792089237316195423570985008687907853269984665640564039457584007913129639934", 10)
+	require.True(t, ok)
```

Comment:
Can you use `math.MaxUint64` and `math.MaxUint64 - 1` values and convert it into a string? It would be more readable than this.

Reply:
Changed it a bit, to be somewhat clearer:
https://github.com/Ethernal-Tech/blade/pull/83/commits/29b07d90517f9d517120636ee1ba759d7aedbede